src/components/BannerApp.vue

```diff
@@ -0,0 +1,103 @@
+<script lang="ts">
+// libraries
+import { defineComponent, computed } from 'vue';
+import { Screen } from 'quasar';
+import { useStorage } from '@vueuse/core';
+
+// types
+import { BannerApp as BannerAppType, ConfigGlobal } from 'components/types';
+
+const rideToWorkByBikeConfig: ConfigGlobal = JSON.parse(
+  process.env.RIDE_TO_WORK_BY_BIKE_CONFIG
+);
+
+export default defineComponent({
+  name: 'BannerApp',
+  props: {
+    banner: {
+      type: Object as () => BannerAppType,
+      required: true,
+    },
+  },
+  setup() {
+    const state = useStorage('ride-to-work-by-bike', { showAppBanner: true });
+    const borderRadius = rideToWorkByBikeConfig.borderRadiusCard;
+    const isHorizontal = computed(() => {
+      return Screen.gt.xs;
+    });
+
+    return {
+      borderRadius,
+      isHorizontal,
+      state,
+    };
+  },
+});
+</script>
+
+<template>
```

Comment:
This component is visually "same" as BannerImage component. Maybe we could we reuse it here (DRY principle).

Reply:
I think I would rather keep them separate.
BannerApp has the function of writing a key (showAppBanner) into the localStorage so that the user can hide it with persistency. This logic seems redundant and confusing on the BannerImage component especially when both are used on the same page together.